samza-core/src/main/java/org/apache/samza/zk/ZkUtils.java

```diff
@@ -300,6 +302,7 @@ public void subscribeToProcessorChange(IZkChildListener listener) {
     zkClient.subscribeChildChanges(keyBuilder.getProcessorsPath(), listener);
   }
 
+  @VisibleForTesting
```

Comment:
Actually, I am just going to remove it. It is not used anywhere but in test classes. It can just be local to the test class.